### PR TITLE
update the code for watsonhealth to watson/health or watson.health

### PR DESCRIPTION
**Pull request by prb112**
changes #486

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Comment by prb112** [comment]
```
[INFO] Tests run: 288, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 176.988 s - in TestSuite
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 288, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:01 min
[INFO] Finished at: 2019-09-03T09:58:33-04:00
[INFO] ------------------------------------------------------------------------
```

Integration tests and Unit Tests pass

File: fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java [hunk 1]
```diff
@@ -155,9 +155,9 @@
 
     private static final String FHIR_SERVER_NAME = "IBM Watson Health Cloud FHIR Server";
     private static final String FHIR_COPY_RIGHT = "(c) Copyright IBM Corporation 2018, 2019";
-    private static final String EXTENSION_URL = "http://ibm.com/watsonhealth/fhir/extension";
-    private static final String BASIC_RESOURCE_TYPE_URL = "http://ibm.com/watsonhealth/fhir/basic-resource-type";
-    private static final String REQUEST_HEADER_URL = "http://www.ibm.com/watsonhealth/fhir/extensions/http-request-header";
+    private static final String EXTENSION_URL = "http://ibm.com/watson/health/fhir/extension";
+    private static final String BASIC_RESOURCE_TYPE_URL = "http://ibm.com/watson/health/fhir/basic-resource-type";
```
Comment (lmsurpre) [inline fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: What is this one used for?

File: docs/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Watson Health FHIR Server version 2.1 was developed under the Watson Health.
 
 ## 2.1 Installing a new server
 1.	To install the FHIR server, first obtain the installation package `fhir-install-<version>.zip`.
-You can download the package from the FHIR server team's [Artifactory server](https://na.artifactory.swg-devops.com/artifactory/webapp/#/artifacts/browse/simple/General/wh-fhir-server-releases-maven-local/com/ibm/watsonhealth/fhir/fhir-install).
+You can download the package from the FHIR server team's [Artifactory server](https://na.artifactory.swg-devops.com/artifactory/webapp/#/artifacts/browse/simple/General/wh-fhir-server-releases-maven-local/com/ibm/watson/health/fhir/fhir-install).
```
Comment (lmsurpre) [inline docs/FHIRServerUsersGuide.md]: this whole section needs to be removed as it points to internal artifacts inside IBM

Comment (prb112) [reply docs/FHIRServerUsersGuide.md]: right - do you want me to remove?

File: fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java (hunk 1 above)
Comment (prb112) [reply fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: EXTENSION_URL is used in the addExtensionElements for the Capability Statement

BASIC_RESOURCE_TYPE_URL
doSearch > createBasicCodeSearchParameter uses it as a valueSystem for the 'code' type.
*** this is for virtual resources 

Not sure about REQUEST_HEADER_URL

```
// If we see an entry for the "http-request-header" URL, then parse the value.
                    if (REQUEST_HEADER_URL.equals(url)) {
```

File: docs/FHIRServerUsersGuide.md (hunk 2 above)
Comment (lmsurpre) [reply docs/FHIRServerUsersGuide.md]: yes please

Comment (lmsurpre) [reply docs/FHIRServerUsersGuide.md]:
```suggestion
The Maven build creates the zip package under `fhir-install/target`. Alternatively, releases will be available from the [Releases tab](https://github.com/ibm/fhir/releases).
```

Comment (lmsurpre) [reply docs/FHIRServerUsersGuide.md]: also, the prevous line should say `fhir-server-distribution.zip` and not `fhir-install-<version>.zip`

File: fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java (hunk 1 above)
Comment (lmsurpre) [reply fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: BASIC_RESOURCE_TYPE_URL and REQUEST_HEADER_URL should get deleted.  We can do that now or in a separate issue / PR.

**Comment by prb112** [comment]
@lmsurpre should be all set.  Thanks, Paul

File: fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java (hunk 1 above)
Comment (prb112) [reply fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: I suggest a separate Issue.  do you want me to raise?

**Comment by prb112** [comment]
Updated with the latest changes 

```
[INFO] Tests run: 288, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 152.225 s - in TestSuite
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 288, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:36 min
[INFO] Finished at: 2019-09-04T07:55:45-04:00
[INFO] ------------------------------------------------------------------------
```

File: fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java (hunk 1 above)
Comment (lmsurpre) [reply fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: sure

Comment (prb112) [reply fhir-server/src/main/java/com/ibm/watson/health/fhir/server/resources/FHIRResource.java]: I ended up merging it here.

File: docs/FHIRServerUsersGuide.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ Watson Health FHIR Server version 2.1 was developed under the Watson Health.
 
 ## 2.1 Installing a new server
 1.	To install the FHIR server, first obtain the installation package `fhir-install-<version>.zip`.
```
Comment (lmsurpre) [inline docs/FHIRServerUsersGuide.md]:
```suggestion
1.	To install the FHIR server, first obtain the installation package `fhir-server-distribution.zip`.
```

File: fhir-server-test/src/test/java/com/ibm/watson/health/fhir/server/test/ServerSpecTest.java [hunk 4]
```diff
@@ -522,7 +522,7 @@ public void testSearchInvalidResourceType() {
         WebTarget target = getWebTarget();
         Response response = target.path("NotAResourceType").queryParam("notasearchparameter", "foo").request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
         assertResponse(response, Response.Status.BAD_REQUEST.getStatusCode());
-        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The virtual resource types feature is not enabled for this server");
+        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The resource type 'NotAResourceType' is not allowed.");
```
Comment (lmsurpre) [inline fhir-server-test/src/test/java/com/ibm/watson/health/fhir/server/test/ServerSpecTest.java]: I assume its right.  Excited to get our integration tests running as part of PRs again so its easier to verify.

File: build/gen-code.sh [hunk 5]
```diff
@@ -11,6 +11,6 @@ mvn clean package install -f ../fhir-tools/pom.xml
 
 if [[ $? -eq 0 ]]
 then 
-    mvn com.ibm.watsonhealth.fhir:fhir-tools:generate-model -f ../fhir-model/pom.xml
+    mvn com.ibm.watson.health.fhir:fhir-tools:generate-model -f ../fhir-model/pom.xml
```
Comment (lmsurpre) [inline build/gen-code.sh]: this is just a test
```suggestion
    mvn com.ibm.watson.health.fhir:fhir-tools:generate-model -f ../fhir-model/pom.xml
```

**Pull request merged**